client/src/leap/soledad/client/crypto.py

```diff
@@ -57,7 +57,7 @@ def _assert_known_encryption_method(method):
         raise crypto.UnknownEncryptionMethodError
 
 
-def encrypt_sym(data, key, method):
+def encrypt_sym(data, key, method=crypto.EncryptionMethods.AES_256_CTR):
```

Comment:
The different encryption methods were added a long time ago when we started thinking about encryption possibilities. I think we can consider these as leftover code and remove it for now, if you think it clarifies what the current code is doing.